modrole.py

```diff
@@ -29,25 +29,31 @@ async def wrapped(*args):
             if len(modroles) <= 0:
                 await ctx.send('no!!')
                 return
-            
-            
             return await func(*args)
         return wrapped
     return wrapper
 
+def server_owner_only():
+    def wrapper(func):
+        @functools.wraps(func)
+        async def wrapped(*args):
+            ctx = next(a for a in args if type(a) is commands.context.Context)
+            if ctx.author.id == ctx.guild.owner_id:
+                return await func(*args)
+            else:
+                await ctx.send('no!!!!')
```

Comment:
I can think of more informative things to say in this case.

Reply:
cool

Reply:
it's a stylistic decision